configuration/device-tree.md

```diff
@@ -332,10 +310,85 @@ Note that a property is assigned the value `false` by not defining it. Boolean p
 name = <&label>,"property?";
 ```
 
-where `label` and `property` are replaced by suitable values. Boolean parameters can cause properties to be created or deleted.
+where `label` and `property` are replaced by suitable values.
+
+Inverted booleans invert the input value before applying it in the same was as a regular boolean; they are declared similarly, but use `!` to indicate the inversion:
+
+```
+name = <&label>,"property!";
+```
 
+Boolean parameters can cause properties to be created or deleted, but they can't delete a property that already exists in the base DTB.
+
+#### 2.2.4: Byte string parameters
 <a name="part2.2.4"></a>
-#### 2.2.4 Overlay/fragment parameters
```

Comment:
I guess this shows the down-side of using numbered sections (instead of textual names) as anchor-links - any external pages that were previously linking directly to "Overlay/fragment parameters" will now be linking to "Byte string parameters" instead :wink:

Reply:
I'm switching them back...